pkg/kubelet/dockertools/manager.go

```diff
@@ -263,6 +273,18 @@ func (sc *reasonInfoCache) Get(uid types.UID, name string) (reasonInfo, bool) {
 	}
 }
 
+func (dm *DockerManager) isDockerAPIVersionCompatible(input string) bool {
```

Comment:
This is a very general function, it just compares two versions (current version >=input) and returns true/false. I dont think so it should be tied to a specific name.